platforms/ide/ide-native/src/main/java/org/gradle/ide/visualstudio/VisualStudioSolution.java

```diff
@@ -56,27 +55,22 @@ public interface VisualStudioSolution extends Named, Buildable, IdeWorkspace {
     /**
      * Configuration for the generated solution file.
      */
-    @Internal
```

Comment:
This type is part of the mutable model and is no longer used (or should be used) as a @Nested bean.